### ISSUE TITLE
WCAG 2.1 AA Accessibility: Setup managed instance for accessibility testing and future audits

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#31246]
## Description

We want to ensure that we have a dedicated instance we can use for automated accessibility testing and any future comprehensive audits.

This is to ensure:
1. We have a stable, controlled environment which we can run accessibility audits against
2. We can easily create demo accounts that our accessibility tooling can safely access. E.g. for auditing admin functionality
3. We can provide TGPi with a stable environment that they can use to complete a full accessibility audit

**Comment by umpox** [comment]
Current status:

Have enquired with #cloud-devops about the possibility of this: https://sourcegraph.slack.com/archives/C02KX975BDG/p1647352381453499

Haven't got a response yet - need to follow up

**Comment by sourcegraph-bot-2** [comment]
Heads up @sourcegraph/cloud-devops - the "team/devops" label was applied to this issue.

**Comment by michaellzc** [comment]
it's done
